prime-angular-frontend/src/app/modules/adjudication/shared/components/review-status-content/review-status-content.component.html

```diff
@@ -29,6 +29,14 @@ <h2 class="h3 mb-4">
           </span>
           <p class="pl-2">
             {{ reason.note }}
+            <ng-container *ngFor="let match of reason.potentialMatches; let isLast=last">
+              <a href=#
+                 (click)="onRoute([match, AdjudicationRoutes.ENROLLEE_OVERVIEW], $event)">
+                {{ DISPLAY_ID_OFFSET + match }}
+              </a>
+              {{ isLast? '.' : ', ' }}
```

Comment:
the period just looks off, and can be dropped as the other notes don't use them either.

Reply:
removed